### PR TITLE
added models and seeds

### PR DESCRIPTION
**Pull request by AnthonyBonanno**
added a basic user model, and a post model (for posts made by users)

added seed for both models (json files)

added User.hasMany and Post.belongsTo in models/index.js

added code into seed.js (with a random number generator for user_id whenever a post is created)

### CODE REVIEW

**Review by Hyne-OS1 — APPROVED** [review]
all model data looks good, pushing to main

File: models/index.js [hunk 1]
```diff
@@ -1,3 +1,17 @@
 // this index.js file is only responsible for defining and configuring the associations between different models in the application
 // eg user.hasMany(Projects, {}), establish foreing key relations
-// module.exports
```
Comment (Hyne-OS1) [inline models/index.js:3]: typo error

**Review by Hyne-OS1 — APPROVED** [review]

**Review by Hyne-OS1 — APPROVED** [review]

**Review by Hyne-OS1 — APPROVED** [review]

File: models/index.js (hunk 1 above)
Comment (AnthonyBonanno) [reply models/index.js:3]: looking good

File: seeds/seed.js [hunk 2]
```diff
@@ -1,9 +1,35 @@
-// seeds file wil use both sequelize package and require root path to config connection.js
```
Comment (Hyne-OS1) [inline seeds/seed.js:1]: resolved

File: seeds/seed.js [hunk 3]
```diff
@@ -1,9 +1,35 @@
-// seeds file wil use both sequelize package and require root path to config connection.js
+// seeds file will use both sequelize package and require root path to config connection.js
 
 // const variable to grab user data. require models file location.
 
 // const userdata "example". these const variables will grab "example.json file". json files are populated from the model data
 // "const userData = require('./userData.json');"
 
 
-// const variable ---> async await function ---- seedDatabase(); at end
```
Comment (Hyne-OS1) [inline seeds/seed.js:9]: resolved

File: untils/helpers.js [hunk 4]
```diff
@@ -6,5 +6,6 @@ module.exports = {
     format_date: (date) => {
       // Format date as MM/DD/YYYY
       return date.toLocaleDateString();
-    }
```
Comment (Hyne-OS1) [inline untils/helpers.js:9]: resolved

**Review by Hyne-OS1 — APPROVED** [review]
resolved

**Review by eddiespag-hetti — APPROVED** [review]

**Review by Hyne-OS1 — APPROVED** [review]

**Pull request merged**